src/main/java/me/rayzr522/jsonmessage/JSONMessage.java

```diff
@@ -905,16 +1007,47 @@ public void setOnHover(MessageEvent onHover) {
         /**
          * @return The color
          */
+        public String getColorValue() {
+            return color;
+        }
+
+        /**
+         * @return The color
+         */
+        @Deprecated
         public ChatColor getColor() {
+            if(this.color == null || this.color.isEmpty())
+                return ChatColor.WHITE;
```

Comment:
hmmm. is this actually semantically the same? I believe the original [just returned null for null values](https://github.com/Rayzr522/JSONMessage/blob/master/src/main/java/me/rayzr522/jsonmessage/JSONMessage.java#L909), would prefer to keep it the same

otherwise this method is perfect, will maintain enough compatibility to not break with compiler errors but for any new code using hex values they should be using the new `getColorValue()` method anyways :)

Reply:
Would the new method be renamed in the future, once this is removed?
I mean the name still makes sense but `getColor` also is stimple.

Reply:
possibly, although we might have to (again) go through a two-step process, first replacing `ChatColor getColor()` with `String getColor()` and marking `String getColorValue()` as deprecated, before eventually totally removing `String getColorValue()` but not sure.

might simply handle this in a breaking major version rewrite, since I think there are some other overall improvements that could be made by a total rewrite. this is a worry of the future, tho :)

Reply:
Agree on this.